.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!--
+
+Read our pull request guide:
+https://sonarwhal.com/docs/developer-guide/contributing/pull-requests.html
+
+For the following items put an "x" between the square brackets
+(i.e. [x]) if you completed the associated item.
+
+-->
+
+## Pull request checklist
+
+Make sure you:
+
+- [ ] Signed the [Contributor License Agreement](https://cla.js.foundation/sonarwhal/sonar)
```

Comment:
Maybe this should be just a comment and not an item? The bot is going to get it automatically and update once it is done.

Reply:
> The bot is going to get it automatically and update once it is done.

My thinking was to have all the items just in one place. 

> The bot is going to get it automatically and update once it is done. 

We can also just remove it.

Reply:
> We can also just remove it.

We can leave it so they know they have to sign something. Some times people just close the browser and don't see anything else.